### PR TITLE
Label Python external SchemaTransform with its URN

### PR DESCRIPTION
**Pull request by ahmedabu98**
Doing this so that we can identify the transform URN when looking at a pipeline DAG.

### CODE REVIEW

File: sdks/python/apache_beam/transforms/external.py [hunk 1]
```diff
@@ -414,7 +414,7 @@ def __init__(
 
   def expand(self, pcolls):
     # Expand the transform using the expansion service.
-    return pcolls | ExternalTransform(
+    return pcolls | self._payload_builder._identifier >> ExternalTransform(
```
Comment (chamikaramj) [inline sdks/python/apache_beam/transforms/external.py:417]: Do possible characters and format for payload builder identifier and PTransform labels match ?

Comment (chamikaramj) [inline sdks/python/apache_beam/transforms/external.py:417]: Let's not use private variable "_identifier" here. (either use something else or add a public property.

Comment (ahmedabu98) [reply sdks/python/apache_beam/transforms/external.py:417]: I can't find documentation on what the standard is for PTransform labels, but I tried the Storage API xlang wrapper with URN `beam:schematransform:org.apache.beam:bigquery_storage_write:v2` and it shows up fine on the Dataflow DAG: 
image

**Comment by ahmedabu98** [comment]
stop reviewer notifications

File: sdks/python/apache_beam/transforms/external.py (hunk 1 above)
Comment (chamikaramj) [reply sdks/python/apache_beam/transforms/external.py]: Also, I think Dataflow might fail if you use the same label twice (i.e. if you use that same SchemaTransform twice in a pipeline). Can you check that ?

Comment (ahmedabu98) [reply sdks/python/apache_beam/transforms/external.py]: I tried the following on Dataflow and it worked fine:
with beam.Pipeline(options=opts) as p:
  input = p | beam.Create([{"hi": "hello"}])
  input | "write1" >> beam.io.WriteToBigQuery(table="...1",
                             method="STORAGE_WRITE_API",
                             schema="hi:STRING")
  input | "write2" >> beam.io.WriteToBigQuery(table="...2",
                                  method="STORAGE_WRITE_API",
                                  schema="hi:STRING")

I think it just cares about the top-level transform naming. Our IOs and other composites have consistent naming and it's up to the user to label top level transforms where needed. Without the top-level labels, I get the following error:
`A transform with label "WriteToBigQuery" already exists in the pipeline. To apply a transform with a specified label write pvalue | "label" >> transform`

**Comment by chamikaramj** [comment]
LGTM. Thanks.

**Pull request merged**